src/components/bookDetails/index.tsx

```diff
@@ -13,7 +13,7 @@ import ReportProblem from "./ReportProblem";
 import Head from "next/head";
 import { H1, H2, H3, Text } from "components/Text";
 import MediumIndicator from "components/MediumIndicator";
-import PalaceLogo from "components/PalaceLogo";
+import SimplyELogo from "components/SimplyELogo";
```

Comment:
Assuming that we still have this component.

Reply:
Yup, I never deleted it.